charts/adot-exporter-for-eks-on-ec2/values.schema.json

```
@@ -62,8 +62,7 @@
                     ],
                     "properties": {
                         "repository": {
-                            "type": "string",
-                            "pattern": "^[/a-z0-9-_]+$"
+                            "type": "string"
```

```suggestion
                            "type": "string"
                            "pattern": "^[.-_/a-z0-9]+$"
```
Can you let us know if this pattern works instead? I am not in support in removing the regex entirely.

---

Thanks for your suggestion, with:`"pattern": "^[.-_/a-z0-9]+$"`
I got this error:
```bash
helm install \
  local charts/adot-exporter-for-eks-on-ec2 \
  --set fluentbit.enabled=true \
  --set adotCollector.daemonSet.enabled=true \
  --set adotCollector.sidecar.enabled=true \
  --set fluentbi.image.repository="'" \
  --set adotCollector.image.repository="public.ecr.aws/aws-observability/aws-otel-collector" \
  --set adotCollector.sidecar.image.repository="public.ecr.aws/aws-observability/aws-otel-collector"

Error: INSTALLATION FAILED: values don't meet the specifications of the schema(s) in the following chart(s):
adot-exporter-for-eks-on-ec2:
- fluentbit.image.repository: Does not match pattern '^[.-_/a-z0-9]+$'
- adotCollector.image.repository: Does not match pattern '^[.-_/a-z0-9]+$'
- adotCollector.sidecar.image.repository: Does not match pattern '^[.-_/a-z0-9]+$'
```

I've found regex pattern `^[a-z0-9\.\-_\/]+$`works for `public.ecr.aws/aws-observability/aws-otel-collector`
=> with json escape `"^[a-z0-9\\.\\-_\\/]+$"`